mleacct-sanjoseca.md

```diff
@@ -0,0 +1,8 @@
+# San Jose, CA
+
+### Restaurants
+
+### Local Attractions
+- Great America Theme Park
```

Comment:
@shinton sorry I don't know about "famous/controversial statue downtown" of San Jose.